shell/platform/windows/flutter_windows_view.cc

```diff
@@ -80,19 +90,22 @@ uint32_t FlutterWindowsView::GetFrameBufferId(size_t width, size_t height) {
 void FlutterWindowsView::OnWindowSizeChanged(size_t width, size_t height) {
   // Called on the platform thread.
   std::unique_lock<std::mutex> lock(resize_mutex_);
+  if (!IsValidResize(resize_target_width_, resize_target_height_, width,
+                     height)) {
+    return;
```

Comment:
When the size doesn't change this is fine, but for 0x0 the surface actually is 0x0. Failing to send that information to the engine means will be out of sync with the surface it's drawing to. (I'm not clear on how problematic that could potentially be; e.g., is there potential for memory corruption as there would be in the case of a pixel buffer?)

Unless we're *extremely* confident that there will be no unexpected side-effects (given that this will need to be cherry-picked) I think for now, the safer approach would be to send the 0x0 metrics as we currently do and only bypass the resize state logic+blocking. We can leave a TODO with a pointer to https://github.com/flutter/flutter/issues/65061 to resolve 0x0 handling in general, via something like a new embedder.h API that we call to tell the engine to suspend drawing, rather than doing nothing.